io.github.Archeb.opentrace.yml

```diff
@@ -0,0 +1,71 @@
+app-id: io.github.Archeb.opentrace
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+separate-locales: false
+rename-icon: opentrace
+rename-desktop-file: opentrace.desktop
+command: OpenTrace
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Flatpak
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: opentrace
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+    build-commands:
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 opentrace.png -t /app/share/icons/hicolor/scalable/apps
+      - install -D opentrace.sh /app/bin/OpenTrace
+      - install -Dm644 opentrace.desktop -t /app/share/applications
+      - install -D apply_extra -t /app/bin
+      - install -Dm755 nexttrace.sh /app/bin/
+      - install -D /usr/bin/desktop-file-edit -t /app/bin/
+    sources:
+      - type: extra-data
+        only-arches: [x86_64]
+        filename: linux-x64.tar.gz
+        url: https://github.com/Archeb/opentrace/releases/download/v1.4.2.1/linux-x64.tar.gz
+        size: 39428555
+        sha256: b7937c807b275abb9f036c951ea41adf61e5a8dea39d1e3723b5717aa2537031
+        x-checker-data:
+          type: json
+          is-main-source: true
+          url: https://api.github.com/repos/Archeb/opentrace/releases/latest
+          version-query: .tag_name | sub("^v"; "")
+          url-query: >-
+            .assets[] | select(.name=="linux-x64.tar.gz") |
+            .browser_download_url
+
+      - type: file
+        path: io.github.Archeb.opentrace.metainfo.xml
+
+      - type: file
+        path: opentrace.png
+
+      - type: file
+        path: opentrace.desktop
+
+      - type: file
+        path: nexttrace.sh
+
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - tar -zxf linux-x64.tar.gz
+          - rm linux-x64.tar.gz
+          - mv nexttrace _nexttrace
+          - cp /app/bin/nexttrace.sh nexttrace
+
+      - type: script
+        dest-filename: opentrace.sh
+        commands:
+          - export WEBKIT_DISABLE_COMPOSITING_MODE=1 # fix https://github.com/Archeb/opentrace/issues/22
```

Comment:
this should be set for Nvidia only

Reply:
This is to resolve a crash which also happens on my platform (with AMD APU, and also Nvidia from other users )

Reply:
ok thought it was for the black screen issue with webkit2gtk which affects nvidia only